libvips/foreign/cgifsave.c

```diff
@@ -72,6 +73,8 @@ typedef struct _VipsForeignSaveCgif {
 	int *delay;
 	int delay_length;
 	int loop;
+	int *gct;
+	int gct_length;
```

Comment:
Could we call this `global_colour_table`? Abbreviations in local variables are OK, but I think members should be written out in full, since they start spreading way beyond a small local context.